dix/enterleave.c

```diff
@@ -640,6 +640,8 @@ FixDeviceValuator(DeviceIntPtr dev, deviceValuator * ev, ValuatorClassPtr v,
     case 1:
         ev->valuator0 = v->axisVal[first];
         break;
+    default:
+        break;
```

Comment:
Is this all just to silence compiler warnings, or are these in fact valid cases that should be ignored, i.e. no errors should be reported? Just for my understanding (as someone who knows quote little about the inner workings here).